daisy/step_create_images.go

```diff
@@ -113,6 +113,7 @@ func (ci *CreateImages) run(ctx context.Context, s *Step) DError {
 			e <- newErr("failed to create images", err)
 			return
 		}
+		ci.markCreated()
```

Comment:
Any reason why you use the method here, but in the other steps you set the field directly?

Reply:
Image is the only special resource. It used a 'ImageInterface' as a wrapper because it wants to be compatible with 2 versions: Image & ImageBeta. So I have to add a method to the interface to operate the 'created' field.